tools/validators/ontology_validator/yamlformat/validator/entity_type_manager.py

```diff
@@ -440,3 +440,12 @@ def _MapTypenamesBySubset(self, types_to_include, field_to_typenames,
       typenames_by_subset[field_set].update(typegroup)
 
     return typenames_by_subset
+
+
+  def GetCompleteFieldSetsOI(self) -> dict():
+    """Returns a mapping of complete field lists to EntityType strings"""
+    return self._complete_field_sets_oi
```

Comment:
This must fail if analyze() has not been called, and a test added for that behavior.   (same with other method)